### PR TITLE
Update User Guide (Command Format, CLI,, Summary Table), developer guide use cases and PPP

### PR DESCRIPTION
**Pull request by PangKuangWei**
### User Guide Changes
- Command Format #236 
- CLI #230 
- Summary Table #229 
- Customise Features Section

Note: Order in summary table is changed for tags in advance as @ChongCheeKaiClarence  is changing the order in his PR.

### Developer Guide Changes
- Added Use cases for customise
- Reorder customise order use story in table to be with the other customise stories

### PPP Update
- Copied the first 2 lines of intro from @Eugene-Ong-W-X

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ SoConnect is a **desktop app for managing contacts and tasks**. It aims to help
 
 # Quick start
 
+This section gives you information on how to install SoConnect and how to start using it. SoConnect uses a [Command Line Interface (CLI)](#CLI) which is where [commands](#Command), which are typed in a certain [format](#Command Format), are entered to use SoConnect features.
+
+## Installation Tutorial
+
 1. Ensure you have Java `11` or above installed in your Computer.
```
Comment (ChongCheeKaiClarence) [inline docs/UserGuide.md]: Maybe we could provide a link to a website to install java 11.
```suggestion
1. Ensure you have Java [`11`](https://www.oracle.com/java/technologies/downloads/#java11) or above installed in your Computer.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -21,28 +25,27 @@ SoConnect is a **desktop app for managing contacts and tasks**. It aims to help
 4. Double-click the file to start the app. A GUI similar to the one below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
+## Command Line Interface Tutorial
```
Comment (ChongCheeKaiClarence) [inline docs/UserGuide.md]: Maybe we can give a disclaimer that this is for newbies and provide a skip link for CLI veterans.

**Review by ChongCheeKaiClarence — COMMENTED** [review]
Great work! Just some improvements

File: docs/UserGuide.md (hunk 1 above)
Comment (PangKuangWei) [reply docs/UserGuide.md]: Ok

File: docs/UserGuide.md (hunk 2 above)
Comment (PangKuangWei) [reply docs/UserGuide.md]: Ok

**Pull request closed without merging**